apypie/api.py

```diff
@@ -189,6 +189,8 @@ def http_call(self, http_method, path, params=None, headers=None, options=None,
 
         request = self._session.request(http_method, full_path, **kwargs)
         request.raise_for_status()
+        if request.status_code == 204:  # no content
```

Comment:
```suggestion
        if request.status_code == requests.codes.no_content:
```

Reply:
oh, and can we easily add a test for that?

Reply:
Like this?